pkg/collector/corechecks/embed/process/stub.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !process || (!darwin && !freebsd)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Is this missing a closing `)`? There's an opening ahead of `darwin` but no matching pair.

Reply:
[Looks closed to me](https://github.com/DataDog/datadog-agent/blob/354c6fc1378a197fd7fb41aab744b15d5bfd2046/pkg/collector/corechecks/embed/process/stub.go#L6C25-L6C25)

```
//go:build !process || (!darwin && !freebsd)
                       ^                   ^
```